lib/graphs/digraph.nit

```diff
@@ -0,0 +1,561 @@
+# Implementations for representing directed graphs, also called digraphs.
+#
```

Comment:
Maybe you can add some information about the digraph concept here.

An interesting example can be found here: http://networkx.lanl.gov/reference/classes.digraph.html